src/main/src/main/java/org/geoserver/config/impl/ContactInfoImpl.java

```diff
@@ -188,6 +248,191 @@ public void setOnlineResource(String onlineResource) {
         this.onlineResource = onlineResource;
     }
 
+    @Override
+    public InternationalString getInternationalAddress() {
+        return internationalAddress;
+    }
+
+    @Override
+    public void setInternationalAddress(InternationalString internationalAddress) {
+        if (internationalAddress != null)
+            this.internationalAddress = new GrowableInternationalString(internationalAddress);
+        else this.internationalAddress = null;
+    }
+
+    @Override
+    public InternationalString getInternationalContactFacsimile() {
+        return internationalContactFacsimile;
+    }
+
+    @Override
+    public void setInternationalContactFacsimile(
+            InternationalString internationalContactFacsimile) {
+        if (internationalContactFacsimile != null)
```

Comment:
The same could be done here, the block is repeated over an over.

Reply:
fixed